src/server/enrollment-system.js

```diff
@@ -362,6 +359,10 @@ function veteranToDependentFinancialsCollection(veteran) {
  * @returns {Object} ES system dependentFinancialsCollection message
  */
 function veteranToSpouseFinancials(veteran) {
+  if (!_.includes(['Married', 'Separated'], veteran.maritalStatus)) {
```

Comment:
Dependent financial info will still be returned if they're unmarried, as that comes from the `veteranToDependentFinancialsCollection` function, and the `child-financial` examples still covers that case. This change should continue to set `hasIncomeData` correctly if no income is present, but also not include the `spouseFinancialsList` at all if there is no spouse.